docs/TimelineGuide.md

```diff
@@ -167,20 +182,32 @@ which is still excellent.
 These are guidelines that cactbot tries to follow for timelines.
 
 * add syncs for everything possible
-* always add an Engage! entry, but add syncs in case there's no /countdown
 * if the first boss action is an auto-attack, consider adding a sync for
-  the first auto-attack or at least the first "starts using" line
-* include the command line used to generate the timeline in a comment at the top
-* prefer actions for syncs over rp text, but rp text syncs if that's the only option
-* if you do sync a phase with rp text, add a large window sync for an action
-* use original names for ability text as much as possible
-* loops should use `jump` instead of having previous abilities have large windows
+  the first auto-attack, or at least the first "starts using" line.
+  This will help ensure the timeline is synced correctly after entering combat.
+* include any special command line flags used to generate the timeline in a comment at the top.
+(The ignore-combatant and ignore-ability flags
+are automatically added by the timeline utility for your convenience.)
```

Comment:
This is a thought for another PR, but maybe we should add the other flags, like `-p` or `-it` etc

Reply:
Agreed. I should have thought of that when I made the previous listing updates. I'll keep that in mind over the next few weeks, since those should be relatively simple to just dump in as we did with ignored combatants.